src/main/java/inf112/skeleton/app/Game.java

```diff
@@ -63,6 +63,24 @@ else if(tile.kind == TileKind.hole){
 
     }
 
+    private void dealDamage(){
+        robot.health--;
+        if (robot.health >= 1){
+            msg ("Oof, that will leave a mark. You have " + robot.health + " health remaining!");
+        }
+        if (robot.health <= 0){
+            robot.transform.position = new Vector2(robot.startPos);
+            robot.health = 9;
+            robot.lives--;
+            if (robot.lives >= 1) {
+                msg("Another one bites the dust! You have " + robot.lives + " lives remaining!");
+            }
+            if (robot.lives <= 0){
+                msg("It's over, you're dead! Better luck next time, idiot!");
+            }
```

Comment:
Dette burde flyttes til en Robot.kill eller respawn metode, you'll figure it out. :) Poenget er at det er kode som bare behandler roboten sin state, og ikke er avhengig av noe i Game, så da er det best å ikke ha det i Game.